monitoring/uss_qualifier/scd/data_interfaces.py

```diff
@@ -0,0 +1,31 @@
+from typing import Optional
+from monitoring.monitorlib.typing import ImplicitDict
+from monitoring.monitorlib.scd_automated_testing.scd_injection_api import InjectFlightRequest
+import datetime
+
+class SCDTestIssue(ImplicitDict):
```

Comment:
I would expect this class to look very similar to the [RID Issue](https://github.com/interuss/dss/blob/10f5192c17695566347eba3e52843ce1c48d06dc/monitoring/uss_qualifier/rid/reports.py#L22) and a key field there is [`queries`](https://github.com/interuss/dss/blob/10f5192c17695566347eba3e52843ce1c48d06dc/monitoring/uss_qualifier/rid/reports.py#L59).  Another key field would be which USS the InjectFlightRequest was sent to.  These fields cannot be populated by the simulator, so the simulator cannot store RequiredResults that contain Issues, so this data structure should be something else -- like KnownIssueFields (no "SCD" prefix needed because this data structure is defined in the module `[...].scd.data_interfaces`.

Reply:
Thank you for this, I have now renamed KnownIssueFields included more fields from the RID Issue. I have also added the relevant_requirements to this for the moment, my thinking is that here we would reference the Annex 4 of the regulation. I am not sure we can remove it if it is not required.